stream-chat-android-offline/src/test/java/io/getstream/chat/android/livedata/controller/ChannelControllerImplEventTest.kt

```diff
@@ -96,73 +89,56 @@ internal class ChannelControllerImplEventNewTest {
 
         val userStartWatchingEvent = randomNewMessageEvent(user = user, createdAt = newDate, message = newMessage)
 
-        channelControllerImpl.handleEvent(userStartWatchingEvent)
+        channelController.handleEvent(userStartWatchingEvent)
 
-        Truth.assertThat(channelControllerImpl.toChannel().lastMessageAt).isEqualTo(newDate)
+        Truth.assertThat(channelController.toChannel().lastMessageAt).isEqualTo(newDate)
     }
 
     // New message event
     @Test
     fun `when new message event arrives, messages should be propagated correctly`() {
-        val updatedAt = Date()
         val user = User(id = CURRENT_USER_ID)
         val message = randomMessage(
-            updatedAt = updatedAt,
+            createdAt = Date(1000L),
```

Comment:
It's not logically right according to our current logic to check `updatedAt`. Current logic checks `createdAt` property